### PR TITLE
docs: update some texts

### PR DESCRIPTION
**Pull request by davlgd**
Also reorder some options (`--format` always at the end for example)

### CODE REVIEW

File: bin/clever.js [hunk 1]
```diff
@@ -190,7 +190,7 @@ function run () {
       metavar: 'before',
       aliases: ['until'],
       parser: Parsers.date,
-      description: 'Fetch logs before this date/time (ISO8601 date or duration, positive number in seconds or duration Ex: 1h)',
+      description: 'Fetch logs before this date/time (ISO8601 date, positive number in seconds or duration, e.g.: 1h)',
```
Comment (hsablonniere) [inline bin/clever.js:193]: suggestion: can you also do this on `after`/`since`?

**Review by hsablonniere — CHANGES_REQUESTED** [review]
Thanks, I added one suggestion.

File: bin/clever.js (hunk 1 above)
Comment (davlgd) [reply bin/clever.js:193]: It's ok and rebased

**Review by hsablonniere — APPROVED** [review]
LGTM :+1:

**Pull request merged**